### DISCUSSION TITLE
cant resubmit twice, but also no pending review apparent?

### DISCUSSION THREAD

**Discussion by laurinVV** [community/community#121017] [GitHub Education]
### Hi everyone,

After repeated issues with providing an acceptable proof and adhering to all the rules for application, it doesnt seem like my application are being reviewed at all.

I ve done "apparently" everything: university email-adress, completed billing information, 2-factor-auth and  astudent ID with date as good proof.
At different parts in the FAQ or docs it tells me my application status can be viewed:

> You can check the status of your application [here](https://education.github.com/discount_requests/pack_application). If your application is still pending review then it will appear at the top of the sidebar to the right of the page. If your application has already been approved then you will be redirected to your GitHub Education benefits.

This sites shows no status, not even a sidebar and asks me to upload proof, but if I do it tells me:
> Your application cannot be reviewed until you fix the following: Your uploaded proof has already been used previously.

I am at a loss, as to how to proceed from here. 

Thanks in advance for any help.

**Comment by eliz-amst** [comment]
Your application is being under review, the more you apply the longer it takes to review and approve, it usually shows you when to expect the decision, for example expect to receive an answer before April 30 etc, you can still submit another applications but if you are submitting the same one, and upload same proof as before (the one you used and it was rejected) it won't let you upload it

**Reply by eliz-amst** [reply]
simply wait for them to review the first application you already submitted, might takes couple of weeks:(

**Comment by samus-aran** [comment]
Hi @laurinVV 👋

It looks like you are still needing to verify your academic status. 

> Your application cannot be reviewed until you fix the following: Your uploaded proof has already been used previously.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

I also suggest you update your billing info (both first and last name) so that the application picks this up. 👍
